test/tests/functional/pbs_multi_sched.py

```diff
@@ -715,7 +715,13 @@ def test_list_multi_sched(self):
 
         self.server.manager(MGR_CMD_LIST, SCHED, id="sc1")
 
-        self.server.manager(MGR_CMD_LIST, SCHED, id="sc2")
+        dir_path = '/var/spool/pbs/sched_dir'
```

Comment:
Use os.path.join(), don't use direct paths.

Reply:
This Line was directly copied from existing mainline code 
https://github.com/PBSPro/pbspro/blob/master/test/tests/functional/pbs_multi_sched.py#L59

Anyways I will change in both places